pyvista/plotting/render_window_interactor.py

```diff
@@ -975,11 +977,13 @@ def enable_terrain_style(self, mouse_wheel_zooms=False, shift_pans=False):
 
         Parameters
         ----------
-        mouse_wheel_zooms : bool, default: False
+        mouse_wheel_zooms : bool, default: True
             Whether to use the mouse wheel for zooming. By default
             zooming can be performed with right click and drag.
+            Pass a float value for to control the zoom factor,
+            default is 1.05.
 
-        shift_pans : bool, default: False
+        shift_pans : bool, default: True
```

Comment:
Could you add a `.. versionchanged::` directive? It would help clarify when this change was introduced.